var/GNIS_data_import.py

```diff
@@ -172,13 +181,12 @@ def main():
     args = docopt(__doc__, version="v0.0.1")
     db = database.db_from_config(args["--section"])
 
-    with open(args["PLACES_FILE"], "r") as place_file:
-        header_line = (
-            place_file.readline().strip().decode("utf-8-sig")
-        )  # Files downloaded from geonames.usgs.gov are UTF8-BOM
+    # Files downloaded from geonames.usgs.gov are UTF8-BOM
+    with io.open(args["PLACES_FILE"], encoding="utf=8=sig") as place_file:
```

Comment:
Not sure if this is a mistake or some alias convention that I'm unaware of.  Should this be this?
```suggestion
    with io.open(args["PLACES_FILE"], encoding="utf-8-sig") as place_file:
```

Reply:
Yep that was a typo I missed. Oops.